source/Halibut/Transport/SecureListener.cs

```diff
@@ -143,16 +132,16 @@ void WaitForPendingConnectionOrCancellation()
                         }
 
                         var client = listener.AcceptTcpClient();
-#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
                         HandleClient(client);
-#pragma warning restore CS4014
                         numberOfFailedAttemptsInRow = 0;
                     }
-                    catch (SocketException e) when (e.SocketErrorCode == SocketError.Interrupted)
+                    catch (SocketException ex) when (ex.SocketErrorCode == SocketError.Interrupted)
                     {
+                        log.WriteException(EventType.Error, "Socket Exception accepting TCP client", ex);
```

Comment:
Be careful here - we dont want noisy logs with stack traces (to hit production code). We've already got too many spots where we see icky stack traces in logs (and tes toutput)